hub/models.py

```diff
@@ -817,6 +819,17 @@ def get_postcode_data(self) -> Optional[PostcodesIOResult]:
 
         return self.postcode_data
 
+    def save(self, *args, **kwargs):
+        if self.phone:
+            try:
+                self.phone = validate_and_format_phone_number(
+                    self.phone, default_countries()
```

Comment:
This is using the global `default_countries` method, but we really want to use the `ExternalDataSource.countries` value that relates to this `GenericData` record instead.